spec/views/item_seasons/show.html.erb_spec.rb

```diff
@@ -3,8 +3,8 @@
 require 'rails_helper'
 
 RSpec.describe 'item_seasons/show', type: :view do
-  let!(:item_season_produces) { ProduceItem.create(name: 'carambola', category: 'fruit') }
-  let!(:item_season) { ItemSeason.create(produce_item_id: item_season_produces.id, month_index: 1, country_code: 'br') }
+  let!(:item_season_produces) { ProduceItem.create(name: 'morango', category: 'fruit') }
```

Comment:
**question:** why change from carambola to morango?

Reply:
I think I commented on this superficially, but while I was trying to write the helper test for produce_items_helper, I used before_all (I think?). And that seems to have added the fruits I tested to the DB. So now I get a `RecordNotUnique` error.

I just changed the name for a fast simple solution, but I should probably look into this. WDYT?

<img width="484" alt="Screenshot 2022-11-09 at 14 51 54" src="https://user-images.githubusercontent.com/87862340/200903902-8ce4c98f-a70a-4990-85d3-4f7d22dd94a7.png">

Reply:
So I looked around a bit, could this be a Database cleaner/Rspec configuration issue? I found this interesting, albeit old StackOverflow post: [rails-how-to-ensure-clean-state-for-test-database](https://stackoverflow.com/questions/32028909/rails-how-to-ensure-clean-state-for-test-database)

So I looked at the [database cleaner docs](https://github.com/DatabaseCleaner/database_cleaner#rspec-example), and I think this could be in the right direction. WDYT?

Reply:
So I added the config here:
![Screenshot 2022-11-09 at 15 49 46](https://user-images.githubusercontent.com/87862340/200915720-f79f9ff1-19b2-42e5-bc86-18ac79b96d39.png)

Now It seems like I'm able to use 'carambola' again, AND I'm getting the failure we got in the workflows. So this seems to have fixed both those issues?

Reply:
Yeap :D. That's also the correct approach when setting up tests. For some reason, I thought we were already doing that. Sorry for that.